.travis.yml

```diff
@@ -2,14 +2,15 @@ sudo: false
 
 language: generic
 
-before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
-cache:
-  directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-    - $HOME/.m2/
+## Sometimes, the cache gets corrupted.  So don't cache.
```

Comment:
@mernst These changes still show up here, even though that changeset was pulled in from master.
Is there something to do to un-confuse github?